### PR TITLE
add more machine-readable data about the license

### PR DESCRIPTION
**Pull request by marado**
This small change is meant to make LibreJS happy.

Closes: #1273

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -22,8 +22,9 @@ module.exports = function(grunt) {
       dist: {
         options: {
           processContent: function(content) {
-            return grunt.template.process('/**!\n\n @license\n <%= pkg.name %> v<%= pkg.version %>\n\n<%= grunt.file.read("LICENSE") %>\n*/\n')
-                + content;
+            return grunt.template.process('/**!\n\n @license magnet:?xt=urn:btih:d3d9a9a6595521f9666a5e94cc830dab83b65699&dn=expat.txt Expat\n <%= pkg.name %> v<%= pkg.version %>\n\n<%= grunt.file.read("LICENSE") %>\n*/\n')
```
Comment (lawnsea) [inline Gruntfile.js]: Where does this magical value come from?

**Comment by marado** [comment]
Oops, yes, I should have said that...

It comes from https://www.gnu.org/software/librejs/free-your-javascript.html
.

**Comment by lawnsea** [comment]
@marado Ok, thanks. Do we also need `/* @license-end */`?

**Comment by marado** [comment]
That's how I read the documentation, yes.

**Comment by nknapp** [comment]
I just was a bit confused, because there is no explicit magnet-link for the MIT-license on the page [on the libreJS-page](https://www.gnu.org/software/librejs/free-your-javascript.html). It just says:

> Please note: When people speak of the "MIT license" they mean either the X11 license or the expat license. Please see which license the code uses, and label it accordingly.

But the [expat-license](http://directory.fsf.org/wiki/License:Expat) is what's in Handlebars' LICENSE.md file, so everything is fine.

I would like to merge the PR. I would normally do 'squash and merge' or 'rebase and merge' but that would make me the author of the commit (I'm not sure with 'rebase and merge'). Could you rebase the branch onto of master so that I can merge it?

**Comment by marado** [comment]
Since this is only one commit, there's nothing really to squash, and rebase and merge *should not* create a new commit (thus not crunching the authorship of the patch). So feel free to go ahead and 'rebase and merge'. If in any case github ends up munching the authorship, well, I won't loose any sleep over it either :-)

**Comment by marado** [comment]
BTW, the magnet links *are* on that page, check section 3.2.2.1 (List of magnet links for valid licenses).

**Comment by nknapp** [comment]
You're right. It's still yours.

Yes, the magnet links *are* on the page, yes, but there is no link for MIT, just for "expat" and "X11".
I just read "expat"-license for the first time, so I wanted to look.

**Comment by marado** [comment]
Oh, I see.
"MIT License" doesn't really refer to one particular license: the term is used for several (slightly different) licenses, so the term 'expat' is used for the particular variant used in Handlebars.
More info, if you're curious: https://en.wikipedia.org/wiki/MIT_License#Variants

**Pull request merged**